cli.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const pify = require('pify');
+
+const osmGeoJson = require('.');
+
+function stringify(geojson, beautify) {
+  if (beautify) {
+    return JSON.stringify(geojson, null, 2);
+  }
+  return JSON.stringify(geojson);
+}
+
+function help() {
+  const help = ` Usage
+    $ osm-geojson <osmId>
+  Options
+    -p --pretty                   To beautify the output.
+    -l  --list                    To provide a list of osmId.
+  Examples
+    $ osm-geojson 365331
+    $ osm-geojson -p 365331
+    $ osm-geojson -p 365331 >> filename.geojson
+    $ osm-geojson -l 365331:ita 148838:usa
+`;
+  console.log(help);
+
+  return Promise.resolve();
+}
+
+function list(osmidmap, beautify) {
+  const promises = [];
+
+  for (const country of Object.keys(osmidmap)) {
+    promises.push(
+      osmGeoJson
+        .get(osmidmap[country])
+        .then(geojson =>
+          pify(fs.writeFile)(country + '.geojson', stringify(geojson, beautify))
+        )
+    );
+  }
+
+  return Promise.all(promises);
+}
+
+function get(osmid, beautify) {
+  return osmGeoJson.get(osmid).then(geojson => {
+    console.log(stringify(geojson, beautify));
+  });
+}
+
+function run() {
+  const argv = {_: []};
+
+  // Parse flags
+  process.argv.slice(2).forEach(arg => {
+    switch (arg) {
+      case '-p':
+      case '--pretty-print':
+        argv.beautify = true;
+        break;
+      case '-l':
+      case '--list':
+        argv.list = true;
+        break;
```

Comment:
I'd explicitly add `-h` and `--help` to print the help message. -h is pretty standard to print the program usage.

Reply:
I just forgot them, you're right.